data/json/items/comestibles/med.json

```
@@ -1677,6 +1677,24 @@
     "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
     "charges": 10,
     "stack_size": 10
+  },
+    {
+    "id": "sugar_pills",
```

```suggestion
    "id": "homeopatic_pills",
```
It's a detail but why not just give it the same id as the name for clarity?

---

Changed names to `homeopathic_pills`.